roles/observe_chargeback/tasks/chargeback_tests.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Get status of all CloudKitty rating modules
+  ansible.builtin.command:
+    cmd: "{{ openstack_cmd }} rating module list"
+  changed_when: false
+  register: module_list
+
+- name: TEST Validate CloudKitty module states
+  ansible.builtin.assert:
+    that:
+      - "'noop' in module_list.stdout and 'True' in (module_list.stdout_lines | select('search', 'noop') | first)"
+      - "'hashmap' in module_list.stdout and 'True' in (module_list.stdout_lines | select('search', 'hashmap') | first)"
+      - "'pyscripts' in module_list.stdout and 'False' in (module_list.stdout_lines | select('search', 'pyscripts') | first)"
+    fail_msg: "CloudKitty module validation FAILED. Module states are not as expected."
+    success_msg: "SUCCESS: CloudKitty modules (noop=True, hashmap=True, pyscripts=False) are configured correctly."
```

Comment:
This is overkill. We don't need to verify the state of the modules except to say that they're present, and enable/disable them if we need a particular one.